inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceServiceImpl.java

```diff
@@ -267,6 +267,21 @@ public Boolean delete(Integer id, String operator) {
         return true;
     }
 
+    @Override
+    public Boolean forceDelete(String groupId, String streamId, String operator) {
+        LOGGER.info("begin to force delete source for groupId={} and streamId={} by user={}",
+                groupId, streamId, operator);
+        Preconditions.checkNotNull(groupId, ErrorCodeEnum.GROUP_ID_IS_EMPTY.getMessage());
+        Preconditions.checkNotNull(streamId, ErrorCodeEnum.STREAM_ID_IS_EMPTY.getMessage());
+
+        sourceMapper.updateByRelatedId(groupId, streamId, SourceStatus.SOURCE_DISABLE.getCode());
```

Comment:
Suggest printing the affected rows into the log.

Reply:
Done.